### PR TITLE
Add SpriteLab entrypoint

### PR DESCRIPTION
**Pull request by ajpal**
Third attempt at creating this entrypoint (#29550 was reverted by #29707, then #29731 was reverted by #29790)

This PR addresses the issue with #29731 by adding a case for `appOptions.app === 'spritelab'` in all of the places we check `appOptions.app === 'gamelab'`. As future work, we'd like to get rid of these hard-coded app type checks at some point.

### CODE REVIEW

File: apps/src/util/unsupportedBrowserWarning.js [hunk 1]
```diff
@@ -17,10 +17,7 @@ export var checkForUnsupportedBrowsersOnLoad = function() {
       if (isMobileDevice()) {
         if (appOptions.app === 'applab') {
           textDivId = '#applab-unsupported-tablet';
-        } else if (
-          appOptions.app === 'gamelab' &&
-          !appOptions.valueTypeTabShapeMap
```
Comment (jmkulwik) [inline apps/src/util/unsupportedBrowserWarning.js:22]: What was the reason for removing this?

Comment (ajpal) [reply apps/src/util/unsupportedBrowserWarning.js:22]: The goal here was to show the warning for gamelab but not spritelab (`valueTypeTabeShapeMap` is only set for spritelab). Now that spritelab is its own app type, though, this check can just be `appOptions.app === 'gamelab'`

Comment (jmkulwik) [reply apps/src/util/unsupportedBrowserWarning.js:22]: ugh... valueTypeTabShapeMap identifying spritelab. 

Thanks for catching that!

**Review by jmkulwik — APPROVED** [review]
LGTM!

**Pull request merged**